core/src/main/scala/org/bitcoins/core/wallet/utxo/AddressTag.scala

```diff
@@ -0,0 +1,14 @@
+package org.bitcoins.core.wallet.utxo
+
+case class AddressTagType(name: String)
+
+case class AddressTag(tag: String, tagTypeOpt: Option[AddressTagType])
```

Comment:
are either of these meant to be abstract classes?

Reply:
Sadly, no. That was the design in #1307, but that required that we have a separate table for every `AddressTagType` because the table would have no way to know how to map every type. The intention of this design is to have the `AddressTag` and `AddressTagType` just be wrappers for strings so that they can all exist in one table.